### PR TITLE
Перевод '13-Modules'

### PR DESCRIPTION
**Pull request by r37r0m0d3l**

### CODE REVIEW

**Comment by denysdovhan** [comment]
Wooops, seems like we already have this chapter in #25. @victorpavlov and @r37r0m0d3l, please, can you summarize your efforts and update #25 according to this new translation. As I've noticed, this PR has a new sections.

May I count on you?

**Comment by r37r0m0d3l** [comment]
This is draft translation from latest (Jul 28, 2016) commit in modules chapter - [13-Modules.md](https://github.com/nzakas/understandinges6/blob/master/manuscript/13-Modules.md). There is so many differences and additions with current translation so I just doesn't bother to track them. Just pump up my typing skills translating from scratch. **And there's a lot of typos.** I don't have time for them right now. Perhaps in 2017.

**Comment by victorpavlov** [comment]
@denysdovhan @r37r0m0d3l As I see, after I've added this translation in March 2016, there really where many updates in that chapter, and latest update was exactly in Jul 28, 2016, so the version of @r37r0m0d3l is newest and more full i guess.

**Comment by victorpavlov** [comment]
@denysdovhan @r37r0m0d3l  But, as I see now, the translation of @r37r0m0d3l been made by some translator, and not adopted yet. I can try to compare both translations and make some new variant, but not on this week. May be on the next one.

**Comment by r37r0m0d3l** [comment]
That was verbatim (word for word) translation. **You have to fix it.** Also there is no additions to dictionary. A hell of a lot of work for **you** guys.

**Comment by denysdovhan** [comment]
@victorpavlov please, merge this branch to your PR's branch and adjust the translation. Let me know when you do this.

**Comment by victorpavlov** [comment]
@denysdovhan Fine.

**Comment by denysdovhan** [comment]
@victorpavlov any updates? I've published 12-Proxies-and-Reflections today. This chapter is next. Hurry up.

**Comment by victorpavlov** [comment]
@denysdovhan OK, when is deadline?

**Comment by denysdovhan** [comment]
@victorpavlov let's say this Sunday.

**Comment by victorpavlov** [comment]
@denysdovhan Fine. It will be ready.

**Comment by victorpavlov** [comment]
@denysdovhan I made chapter adjustments and pushed updates to my branch. And it's in this PR https://github.com/denysdovhan/understandinges6ua/pull/25/. Thank you.

**Comment by denysdovhan** [comment]
I don't like you haven't saved @r37r0m0d3l's authoring of these changes. But, okay, thank you. Gonna close this PR.

**Comment by victorpavlov** [comment]
@denysdovhan  @r37r0m0d3l Guys, sorry, I did it not on purpose. May you, @denysdovhan, can merge this branch with my, but take my as default, an d we will have @r37r0m0d3l in contributors? Or may be @r37r0m0d3l can make some edits in my translation?
Sorry once again.

**Comment by denysdovhan** [comment]
I believe we shouldn't do unnecessary and @r37r0m0d3l will understand us.
